lib/src/client/channel.dart

```diff
@@ -39,6 +39,11 @@ abstract class ClientChannel {
   /// Initiates a new RPC on this connection.
   ClientCall<Q, R> createCall<Q, R>(
       ClientMethod<Q, R> method, Stream<Q> requests, CallOptions options);
+
+  /// Stream to listen for connection changes
```

Comment:
Maybe: `Stream of connection state changes`.

Reply:
Done in the new commit